static/styles/input_pill.css

```diff
@@ -73,6 +73,17 @@
         }
     }
 
+    &.pill-container-btn {
+        cursor: pointer;
+        padding: 0;
```

Comment:
Is this necessary? Because the default padding should already be 0, except this inherits some padding?

Reply:
Not too sure since this change is ported from a previous PR, but yes it does seem this is necessary to override the inherited padding (of 2px)